### PR TITLE
update obsolete troubleshooting information

### PR DESCRIPTION
**Pull request by DavidVorick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,16 +69,8 @@ Troubleshooting
...
-
-- I'm 100% sure my ports are open, but Sia won't let me announce as a host.
-
-  siad tries to verify your connectivity by pinging your external IPv4 address.
-  This method is sufficient for most people, but for unusual setups it may
-  report false negatives. To override this check, you can "force" the
-  announcement by running `siac host announce [ip:port]`. You can determine
-  your external IP by running `siac gateway` or using a 3rd-party service.
+  Sia currently has support for upnp. While not all routers support upnp, a majority
+  of users should have their ports automatically forwarded by upnp.
```
Comment (lukechampine) [inline README.md]: UPnP needs proper capitalization. This also doesn't seem 80-char wrapped.

**Comment by DavidVorick** [comment]
fixed

**Comment by DavidVorick** [comment]
updated branch, should be ready for merge

**Comment by DavidVorick** [comment]
darn it lol, I updated it and then immediately merged a PR.

**Pull request merged**